src/pages/home.css

```diff
@@ -0,0 +1,212 @@
+#page {
+	margin-left: 40px;
+	margin-right: 40px;
+	display: block;
+	margin-top: 49px;
+}
+
+/* #region Page Header */
+
+#page-header {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+}
+
+#page-header-main {
+	display: flex;
+	align-items: center;
+}
+
+#page-header-main-generic-profile-icon {
+	width: 40px;
+	height: 40px;
+	background-color: orange;
+	border-radius: 50px;
+}
+
+#page-header-main-text {
+	width: 100%;
+	padding-left: 10px;
+
+	font-family: 'Inter';
+	font-style: normal;
+	font-weight: 700;
+	font-size: 39px;
+
+	color: #4b4b4b;
+}
+
+#page-header-button {
+	width: 250px;
+	height: 45px;
+
+	background: #fbfbfb;
+	border: 1px solid #cccccc;
+	border-radius: 10px;
+
+	display: flex;
+	align-items: center;
+}
+
+#page-header-button-icon {
+	position: relative;
+	left: 20px;
+
+	color: #505050;
+}
+
+#page-header-button-text {
+	position: relative;
+	left: 40px;
+
+	font-family: 'Inter';
+	font-style: normal;
+	font-weight: 600;
+	font-size: 17px;
+
+	color: #4b4b4b;
+}
+
+/* #endregion Page Header */
+
+/* #region Page Subheader 1 */
+
+#page-subheader-1-text {
```

Comment:
Since this and subheader-2 are pretty much the same, you could just make a subheader class and then just change the text color in line with tailwind CSS. So 
```
// styles.css
.subHeading {
margin-top: 26px;
	font-family: 'Inter';
	font-style: normal;
	font-weight: 600;
	font-size: 24px;

// home.tsx
.....
<div className="subHeading text-[#a4a4a4]" />
```

Reply:
Yeah it's probably a good idea to make a class just for the sub-header texts.

Why do you suggest to move the color outside of the CSS file?

Reply:
There are multiple subheadings although, for the most part, the styling is all the same besides the text color. By doing it this way you can make one class name for all subheadings, but just change the text color when needed. This way you don't need page-subheading-1-text and page-subheading-2-text for such a small change.